packages/eas-cli/src/credentials/manager/AndroidActions.ts

```diff
@@ -92,25 +92,61 @@ export const fcmActions: ActionInfo[] = [
   },
 ];
 
-export const gsaKeyActions: ActionInfo[] = [
+export const gsaKeyActionsForFcmV1: ActionInfo[] = [
   {
-    value: AndroidActionType.SetUpGsaKey,
-    title: 'Set up a Google Service Account Key',
+    value: AndroidActionType.SetUpGsaKeyForFcmV1,
+    title: 'Set up a Google Service Account Key for Push Notifications (FCM V1)',
     scope: Scope.Project,
   },
   {
-    value: AndroidActionType.CreateGsaKey,
-    title: 'Upload a Google Service Account Key',
+    value: AndroidActionType.UseExistingGsaKeyForFcmV1,
+    title: 'Select an existing Google Service Account Key for Push Notifications (FCM V1)',
+    scope: Scope.Project,
+  },
+  {
+    value: AndroidActionType.GoBackToHighLevelActions,
+    title: 'Go back',
+    scope: Scope.Manager,
+  },
+];
+
+export const gsaKeyActionsForSubmissions: ActionInfo[] = [
+  {
+    value: AndroidActionType.SetUpGsaKeyForSubmissions,
+    title: 'Set up a Google Service Account Key for Play Store Submissions',
+    scope: Scope.Project,
+  },
+  {
+    value: AndroidActionType.UseExistingGsaKeyForSubmissions,
+    title: 'Select an existing Google Service Account Key for Play Store Submissions',
     scope: Scope.Project,
   },
   {
-    value: AndroidActionType.UseExistingGsaKey,
-    title: 'Use an existing Google Service Account Key',
+    value: AndroidActionType.GoBackToHighLevelActions,
+    title: 'Go back',
+    scope: Scope.Manager,
+  },
+];
+
+export const gsaActions: ActionInfo[] = [
+  {
+    value: AndroidActionType.ManageGoogleServiceAccountKeyForSubmissions,
+    title: 'Manage your Google Service Account Key for Play Store Submissions',
+    scope: Scope.Manager,
+  },
+  {
+    value: AndroidActionType.ManageGoogleServiceAccountKeyForFcmV1,
+    title: 'Manage your Google Service Account Key for Push Notifications (FCM V1)',
+    scope: Scope.Manager,
+  },
+  {
+    value: AndroidActionType.CreateGsaKey,
+    title: 'Upload a Google Service Account Key to the Keystore',
```

Comment:
or maybe `Upload a Google Service Account Key to Expo`? A keystore is a type of Android build credential, so best not to confuse folks with that term.
```suggestion
    title: 'Upload a Google Service Account Key',
```